backend/eventloop/application_event_loop/application_event_loop.go

```diff
@@ -114,8 +114,34 @@ func startApplicationEventQueueLoopWithFactory(ctx context.Context, aeqlParam Ap
 
 }
 
+type applicationEventQueueLoopState struct {
+
```

Comment:
Note for reviewer: I moved these out of `applicationEventQueueLoop`, and added some comments, they are otherwise the same as before.

Reply:
Ok, I don't see where these fields are moved out from `applicationEventQueueLoop` at line 65. You'd think that there would be a git diff on this?